Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3.7
```

Comment:
Did the python support version get bumped to 3.8? If so, I'm confused why we didn't bump the image too